pages/common/pio-team.md

```diff
@@ -0,0 +1,32 @@
+# pio team
+
+> Manage PlatformIO teams.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/team/>.
+
+- Create a new team with a description:
+
+`pio team create --description {{description}} {{orgname}}:{{teamname}}`
+
+- Delete a team:
+
+`pio team destroy {{orgname}}:{{teamname}}`
+
+- Add a new member to a team:
+
+`pio team add {{orgname}}:{{teamname}} {{username}}`
+
+- Remove a member from a team:
+
+`pio team remove {{orgname}}:{{teamname}} {{username}}`
+
+- List all teams and their members, the user is part of:
+
+`pio team list`
+
+- List all teams and their members of an organisation:
```

Comment:
```suggestion
- List all members of an organization:
```

Reply:
Isn't that an oversimplification? Your suggeston would be `pio org list`, but here it also lists all teams.

Reply:
Updated.  Does that sound better?